cmd/kubectl-image/import.go

```diff
@@ -54,16 +55,32 @@ var imageimport = &cobra.Command{
 			return err
 		}
 
-		mirror, err := c.Flags().GetBool("mirror")
+		mirror, err := c.Flags().GetString("mirror")
```

Comment:
Why change this from a bool to a string that can only be `"true"` or `"false"`?

Reply:
Let me see if I can explain this: `mirror` and `insecure-source` can be set to one of these values: "true", "false" or "undefined". If "undefined" then we use whatever is defined in the Image as the default for the ImageImport. If it is "true" or "false" instead then we "overwrite" (for that specific import) the values from the "parent Image". Does that make sense?

Reply:
For sake of completeness: https://github.com/shipwright-io/community/pull/57#discussion_r801516122